package.json

```diff
@@ -22,5 +22,8 @@
 		"postversion": "npx liferay-js-publish",
 		"format": "prettier --write \"skins/**/*.css\" \"support/*.js\" \"*.json\" \"*.md\"",
 		"format:check": "prettier --list-different \"support/*.js\" \"*.json\" \"*.md\""
+	},
+	"dependencies": {
+		"jquery": "^3.5.1"
```

Comment:
Don't add a runtime dependency on jQuery. This should be in `devDependencies`.

Reply:
Ok,

Just so you know, this was just a temporary "solution"

It's impossible to `Quail` without `jQuery` (needed by  the `ay11checker` plugin) and this was an attempt of including it.
I think what we should try to do is configure the `a11ychecker` plugin with something else than `Quail` (and hopefully without `jQuery`) if it's possible - if not we should do this correctly.